langwatch/src/components/DashboardLayout.tsx

```diff
@@ -492,9 +500,43 @@ export const DashboardLayout = ({
               </InputGroup>
             </form>
           )}
-          <HStack gap={6} flex={1}>
+
+          <HStack gap={2} flex={1}>
             <Spacer />
-            <HStack gap={4}>
+
+            <Progress.Root
+              defaultValue={0}
+              max={usage.data?.activePlan.maxMessagesPerMonth}
+              value={usage.data?.currentMonthMessagesCount}
+              maxW="150px"
+              colorPalette="orange"
+              width="full"
+            >
+              <HStack>
+                <Link href="/settings/usage" width="150px" cursor="pointer">
+                  <Progress.Label fontSize="xs">
+                    <HStack gap={1}>
+                      Trace Usage
```

Comment:
drop the Trace, just Usage, it's cleaner

<img width="244" height="304" alt="image" src="https://github.com/user-attachments/assets/fc893532-2bf8-48d2-98ba-f25db88eed32" />